docs/eventing/samples/sequence/sequence-terminal/README.md

```diff
@@ -138,8 +139,8 @@ kubectl -n default create -f ./ping-source.yaml
 ### Inspecting the results
 
 You can now see the final output by inspecting the logs of the event-display
-pods. Note that since we set the `PingSource` to emit every 2 minutes, it
-might take some time for the events to show up in the logs.
+pods. Note that since we set the `PingSource` to emit every 2 minutes, it might
+take some time for the events to show up in the logs.
```

Comment:
```suggestion
pods.

**NOTE:** The `PingSource` is set to emit every 2 minutes, so it may
take some time for the events to show up in the logs.
```